src/Hook/ActionShopAccountUnlinkAfter.php

```diff
@@ -56,7 +56,8 @@ public function execute(array $params = [])
         $rsaKeysProvider = $this->module->getService(RsaKeysProvider::class);
         try {
             $rsaKeysProvider->cleanupKeys();
-            $rsaKeysProvider->generateKeys();
+            // FIXME: We MUST generate a new key here to avoid concurrent key re-generation at Presenter display time
```

Comment:
Hello!
Could you explain this? the generateKeys() method is not suitable, but we keep it as a comment?